services/frontend-v3/src/components/searchBar/SearchBar.jsx

```diff
@@ -52,7 +54,12 @@ const SearchBar = () => {
   const handleSearch = (values) => {
     const query = queryString.stringify(values, { arrayFormat: "bracket" });
     const url = `/secured/results?${encodeURI(query)}`;
-    history.push(url);
+    {
+      query !== ""
+        ? history.push(url)
+        : // eslint-disable-next-line no-console
+          console.log("");
+    }
```

Comment:
I would change the ternary operator for a plain old if statement here, making it more readable (and maybe you won't need the eslint disable at the top of the file?)
```suggestion
    if (query !== "") {
      history.push(url);
    }
```
Appart from that, looks good